dietpi/dietpi-software

```diff
@@ -12465,6 +12484,47 @@ _EOF_
 
 		fi
 
+    software_id=181 # Paper
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# Make sure user agrees to the EULA
+			G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+			if G_WHIP_YESNO "Do you agree to the Minecraft EULA found at.
+\nhttps://account.mojang.com/documents/minecraft_eula"; then
```

Comment:
We might want to ask this before even downloading the main jar during first install step? We'll merge install and config steps in the future, so no need to fulfil any strict separation between those 😉.

It could be even asked during interactive selection from dietpi-software menu, where a bunch of other questions and information is prompted based on selection. But would this would not include install via `dietpi-software install 181`, so not sufficient when we want to strictly require the agreement.